docs/reference/filebeat/filebeat-input-gcs.md

```diff
@@ -369,6 +371,95 @@ filebeat.inputs:
     poll_interval: 11m
 ```
 
+### Custom properties [attrib-custom-properties]
+
+Some object properties can be `set` or `overridden` at the input level with the help of certain configuration options. Allowing users to set or override custom object properties provides more flexibility when reading objects from a remote storage where the user might only have read access.
+
+**The supported custom properties are:**
+
+1. [`content_type`](#attrib-content-type)
+2. [`encoding`](#attrib-encoding)
```

Comment:
```suggestion
- [`content_type`](#attrib-content-type)
- [`encoding`](#attrib-encoding)
```

Reply:
We usually reserve numbers for steps in a procedure, and use unordered lists otherwise.

Reply:
Similar comment for the proposed commit message; the list does not need to be ordered, being a set of changes.